### PR TITLE
Added Option to Set default layout from Config

### PR DESCRIPTION
**Pull request by abbasmashaddy72**

### CODE REVIEW

**Comment by awcodes** [comment]
Can you do this so that it reads from the plugin as a modifier that defaults to the config? I'd like to be able to set it at that level with something like CuratorPlugin::defaultListView('table' || 'grid')

**Comment by abbasmashaddy72** [comment]
> Can you do this so that it reads from the plugin as a modifier that defaults to the config? I'd like to be able to set it at that level with something like CuratorPlugin::defaultListView('table' || 'grid')

We Can use Like:
`->defaultView('grid' || 'list')`

**Comment by abbasmashaddy72** [comment]
> Can you do this so that it reads from the plugin as a modifier that defaults to the config? I'd like to be able to set it at that level with something like CuratorPlugin::defaultListView('table' || 'grid')

This is Done Check and let me know if any changes

**Comment by awcodes** [comment]
Thank you. I will look it over as soon as I get a chance. :)

File: config/curator.php [hunk 1]
```diff
@@ -58,4 +58,7 @@
         'display_upload_new' => true,
     ],
     'multi_select_key' => 'metaKey',
+    'table' => [
+        'layout' => 'list',
```
Comment (awcodes) [inline config/curator.php]: Please default this to 'grid' since it is the current default.

File: src/CuratorPlugin.php [hunk 2]
```diff
@@ -103,6 +105,11 @@ public function shouldRegisterNavigation(): ?bool
         return $this->shouldRegisterNavigation ?? config('curator.should_register_navigation');
     }
 
+    public function defaultListView(): ?string
```
Comment (awcodes) [inline src/CuratorPlugin.php]: please rename to 'getDefaultListView' and update where appropriate.

**Review by awcodes — CHANGES_REQUESTED** [review]

**Review by awcodes — APPROVED** [review]

**Pull request merged**